### ISSUE TITLE
Seems a color issue

### ISSUE THREAD

**Issue by tikismoke** [dhmay/jamstats#166]
This is what i get in logs. I can send you the files in FB if need.

```
jamstats version 1.4.1, by Damon May
Mode not specified. Inferred mode pdf
Output filepath not provided. Using input filepath with extension .pdf instead of .json
Writing PDF file...
Using theme white
[2656] Failed to execute script 'jamstats' due to unhandled exception!
Traceback (most recent call last):
  File "seaborn\palettes.py", line 251, in color_palette
  File "matplotlib\colors.py", line 386, in to_rgb
  File "matplotlib\colors.py", line 187, in to_rgba
  File "matplotlib\colors.py", line 262, in _to_rgba_no_colorcycle
ValueError: Invalid RGBA argument: ''

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "jamstats", line 255, in <module>
  File "jamstats", line 251, in __main__
  File "jamstats\plots\plot_together.py", line 93, in save_game_plots_to_pdf
  File "jamstats\plots\plot_together.py", line 67, in make_all_plots
  File "jamstats\plots\basic_plots.py", line 122, in plot
  File "jamstats\plots\plot_util.py", line 104, in make_team_color_palette
  File "seaborn\palettes.py", line 253, in color_palette
ValueError: Could not generate a palette for <map object at 0x000001B17D2267C8>

```

**Comment by dhmay** [comment]
Yes, please send me the JSON. I can put in a defensive check for empty team color strings, regardless, but it'll speed things up if I can verify that it works on your file.

**Comment by dhmay** [comment]
@tikismoke , see if [this](https://github.com/dhmay/jamstats/actions/runs/6701239074) fixes it. (the zipfile build artifact linked at the bottom has a .exe in it)

**Comment by erevrav** [comment]
(I got the identical failure with a game from this weekend & this artifact resolved it. I can send you the json if you'd like & will follow up out of band.)

**Comment by dhmay** [comment]
Thanks, @erevrav ! Glad my fix resolved it.

Do you know what triggered the problem, for you? A new Scoreboard version, maybe, or a team with colors defined oddly?

**Comment by erevrav** [comment]
I thought I'd set up uniform colors in the game (purple and white), but they don't show up when I open the file json inside crg. (some of the other games from that weekend have uniform colors showing up like I'd expect.)

we were running a fork of 2023.4 (specifically https://github.com/katpet/scoreboard/releases/tag/v2023.4_pbt_v0.3 which is just penalty-box-timer specific feature work, so *probably* unrelated?).

**Comment by dhmay** [comment]
Fixed in JamStats 1.4.2.
